getting_started/cli.md

```diff
@@ -380,7 +381,7 @@ Right now the code containing the logic of each of the applications always gets
 
 ## Using `Readline` and `NCurses`
 
-In case we want to build richer CLI applications, there are libraries that can help us. Here we will name two well-known libraries: `Readline` and `NCurses`.
+In case we want to build richer CLI applications, there are libraries that can help us. Here we will name two well-known libraries: `Readline` and` NCurses`.
```

Comment:
```suggestion
In case we want to build richer CLI applications, there are libraries that can help us. Here we will name two well-known libraries: `Readline` and `NCurses`.
```